app/views/settings/email/index.html.erb

```diff
@@ -1,10 +1,13 @@
+<% content_for :back_link do %>
+  <%= render MojForms::BackLinkComponent.new(href: settings_submission_index_path) do %>
```

Comment:
As there seem to be a bit of repetition with this code, maybe can be extracted to a partial that receives 2 locals as customisation, one for the href of the back link, and another for the visually hidden copy. But no biggie!